git-tar/function/ops.go

```diff
@@ -41,32 +41,35 @@ type cfg struct {
 	Frontend *string `json:"frontend,omitempty"`
 }
 
-func parseYAML(pushEvent sdk.PushEvent, filePath string) (*stack.Services, error) {
+func parseYAML(filePath string) (*stack.Services, error) {
 	envVarSubst := false
 	parsed, err := stack.ParseYAMLFile(path.Join(filePath, "stack.yml"), "", "", envVarSubst)
 	return parsed, err
 }
 
 func fetchTemplates(filePath string) error {
-	templateRepos := formatTemplateRepos()
+	templateRepos, err := formatTemplateRepos()
+	if err != nil {
+		return fmt.Errorf("Failed to format template repos: %t", err)
+	}
 
 	for _, repo := range templateRepos {
```

Comment:
We don't return where there are errors, but we used to. Why?

Reply:
I was trying to cater for the scenario where a user has `n` invalid repos, 

If we returned on first error we would have to fix, and re-deploy `n` times. 

Ill re-work this to capture all errors and return a rollup

Reply:
I am happy with failing on the first one.